items.rb

```diff
@@ -0,0 +1,22 @@
+require 'date'
+
+class ClassName
+  def initialize(id, date, archived: true)
+    @id = id
```

Comment:
I'm so sorry for not pointing out this earlier, but I think it's better to set id using Random so that if the id is passed as nill a Random number is assigned to the book. Think of the database. When we load the book from database it will have the id stored with it but when a new book is created by user it would be without id so passing nill to constructor would automatically generate a random id.
Instead of `@id  = id`, I think it's better to use `@id = id.nil? ? Random.rand(1..1000) : id` What do you say? 🤔